nncf/openvino/graph/model_utils.py

```diff
@@ -47,3 +50,13 @@ def remove_fq_from_inputs(model: ov.Model, graph: NNCFGraph) -> ov.Model:
         nodes_queue.extend(graph.get_next_nodes(current_node))
 
     return model_transformer.transform(transformation_layout)
+
+
+def get_input_nodes(nncf_graph: NNCFGraph) -> List[NNCFNode]:
+    """
+    Get all nodes from given nncf_graph that are identified as a input nodes.
+
+    :param nncf_graph: NNCFGraph to work with.
+    :return: Target NNCFGraph input nodes.
+    """
+    return nncf_graph.get_input_nodes() + nncf_graph.get_nodes_by_metatypes([OVReadValueMetatype])
```

Comment:
`ReadValue` is not a true input node. It's a node with the only input as `Constant`.
Concurrence of the `nncf_graph.get_input_nodes` and `model_utils.get_input_nodes` is really confusing.

Reply:
I've renamed it to `get_start_nodes_for_activation_path_tracing`